Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 CODE_DIR=$(shell pwd)/iso
-VERSION=1.0.0
+VERSIONFILE=VERSION
```

Comment:
Why we are placing version to a different file? why can't we have it here and pass this variable across script because anyway scripts are executed from the make.